### PR TITLE
feat: Add Contact organizer feature

### LINKED ISSUE #5811: Implement Organizer Contact Form and Process
Currently if the organizers don't publish their contact in the event listing, the attendees won't be able to contact them even after they purchase the ticket and get the confirmation email. Hence, we need an organizer contact form. 

**How should this work?**

There should be a link in the confirmation email where attendees can reach out to organizers via a contact form. Organizers should then get an email to their email inbox from eventyay, but with the attendee or user email as the reply-contact.

**What is the process**

The goal would be to have a process similar to eventbrite as follows:
As the organizer, I receive an no-reply email within in like 10 minutes from the event system containing the email of the attendee (as they have provided that into the contact form earlier). And this is the end of eventbrite's role. The organizers can choose to contact the attendees directly or to ignore it. 

**What functions do we need for the implementation**
- [ ] Add a **button “Contact Organizers”** next to “Refund button” below ticketing on any event page, e.g. https://eventyay.com/e/3dbaaa50 (the contact button should only be viewable by logged in users)
- [ ] Provide a **contact form** as a pop up following the design of the system.
- [ ] **Send an email** 
  - [ ] to the **owner** and all **organizers** of the event 
     - [ ] show all emails in “To” field, so the organizers can see who received the email
     - [ ] define user (sender) email as a reply, so a response does not go to the eventyay email
  - [ ] Show **“{name of user} via eventyay”** as the the sender (do not show the emails of the organizers in email header), use “no-reply” address and start email as follows: “Hello, you have contacted the organisers of the event {event name}. Below you find a copy of your email. Organisers have received your message and will follow up with you. This is a system message. Please do not reply to this message. Replies are not monitored. Thank you.”
- [ ] Send a **copy of the email to the sender**. Above the copy add the message "This is a copy of your message to the organizers of {event name}. Show eventyay as the sender.
- [ ] **After sending show a message to the sender** “Your message was sent to the organizers of this event. You should receive a copy of the message to your email inbox. The organizers will contact you by email.”

**Examples**
Sample way of contacting organizers from Eventbrite**. In the confirmation email (after completing an order), there is a button to contact the organizer
Screen Shot 2020-07-28 at 19 44 57

Then they will direct us to eventbrite login and where we can fill in a contact form
Screen Shot 2020-07-28 at 19 45 26

Screen Shot 2020-08-03 at 16 24 13

After sending the form. there was a notice saying that organizers will reply in some days.
Screen Shot 2020-08-03 at 11 09 02

### PR DESCRIPTION
**Pull request by sachinchauhan2889**
Fixes #5811 

#### Short description of what this resolves:

**What functions do we need for the implementation**
- [x] Add a **button “Contact Organizers”** next to “Refund button” below ticketing on any event page, e.g. https://eventyay.com/e/3dbaaa50 (the contact button should only be viewable by logged in users)
- [x] Provide a **contact form** as a pop up following the design of the system.
- [x] **Send an email** 
  - [x] to the **owner** and all **organizers** of the event 
     - [x] show all emails in “To” field, so the organizers can see who received the email
     - [x] define user (sender) email as a reply, so a response does not go to the eventyay email
  - [x] Show **“{name of user} via eventyay”** as the the sender (do not show the emails of the organizers in email header), use “no-reply” address and start email as follows: “Hello, you have contacted the organisers of the event {event name}. Below you find a copy of your email. Organisers have received your message and will follow up with you. This is a system message. Please do not reply to this message. Replies are not monitored. Thank you.”
- [x] Send a **copy of the email to the sender**. Above the copy add the message "This is a copy of your message to the organizers of {event name}. Show eventyay as the sender.
- [ ] **After sending show a message to the sender** “Your message was sent to the organizers of this event. You should receive a copy of the message to your email inbox. The organizers will contact you by email.”

### CODE REVIEW

File: app/templates/components/public/ticket-list.hbs [hunk 1]
```diff
@@ -218,6 +218,12 @@
         {{/if}}
       </div>
     </div>
+    <div class="column sixteen wide right floated">
```
Comment (iamareebjamal) [inline app/templates/components/public/ticket-list.hbs:221]: This should only be shown to logged in users AFAIK

Comment (sachinchauhan2889) [reply app/templates/components/public/ticket-list.hbs:221]: If user click on this . Login modal opens

File: app/controllers/public/index.js [hunk 2]
```diff
@@ -116,6 +117,20 @@ export default class IndexController extends Controller {
     this.send('save');
   }
 
+  @action
+  openContactOrganizerModal() {
+    if (!this.session.isAuthenticated) {
+      this.flashMessages.add({
+        message           : 'In order to buy tickets you need to login. If you have not registered yet, please create an account first. Thank you!',
```
Comment (sachinchauhan2889) [inline app/controllers/public/index.js:124]: Message is remaining to changes. Sorry.

**Comment by sachinchauhan2889** [comment]
Validation in also remaining. User can't contact with empty message.

**Comment by mariobehling** [comment]
Ok, I recommend to send unfinished PRs to draft mode and to first go through the points in the issue one by one and resolve them.

**Comment by mariobehling** [comment]
Please move the contact button to the left of the "refund" button.
Screenshot from 2021-01-23 00-56-26

**Comment by mariobehling** [comment]
As we have an option to get a ticket currently for non-verified accounts and it is probably important to mention: Only users with a verified account should be able to contact an organizer. Is this possible?

**Comment by mariobehling** [comment]
Very good. Could you finalize this with the following, please:
[ ] **After sending show a message to the sender** “Your message was sent to the organizers of this event. You should receive a copy of the message to your email inbox. The organizers will contact you by email.”
[ ] * To make it clearer that this is an email message, please add two fields a) "From" [Name and Email of Sender] b) To [Event Organizers]
[ ] * When there is not refund option move the button to the right
[ ] * On the pop up change "Close" -> "Cancel", "Contact" -> "Send Email"
[ ] * The email message does not have line breaks. Please ensure line breaks are adhered to in the message.

Screenshot from 2021-01-25 01-16-52

Screenshot from 2021-01-25 01-04-09

Screenshot from 2021-01-25 01-03-13

**Comment by iamareebjamal** [comment]
About 2. We can show the fields but not make it editable because we already have email of the user and don't want them to send some other emails as reply-to

And we cannot show event organizers name and email as it is private

File: app/templates/components/modals/contact-organizer.hbs [hunk 3]
```diff
@@ -3,18 +3,45 @@
...
-        </div>
-        <button class="ui positive right labeled icon submit right floated button" {{action 'contactOrganizer'}}>
-            {{t 'Contact'}}
-            <i class="checkmark icon"></i>
-        </button>
+    {{#if this.mailSent}}
+        <h4>
+            {{t 'Your message was sent to the organizers of this event.'}}
+            {{t ' You should receive a copy of the message to your email inbox.'}}
+            {{t ' The organizers will contact you by email.'}}
```
Comment (iamareebjamal) [inline app/templates/components/modals/contact-organizer.hbs:10]: No leading space in strings

**Pull request merged**